geode-connectors/src/test/java/org/apache/geode/connectors/jdbc/JdbcDistributedTest.java

```diff
@@ -519,6 +507,7 @@ public void clientGetReadsFromDBWithPdxClassName() throws Exception {
   }
 
   @Test
+  @Ignore("GEODE-4833: Postgres does not support UTF-8 null character as contents of a string")
```

Comment:
Should this ignore be removed since you changed the product to map the null character to NULL?